source/javascripts/components/GuidedOnboarding/GuidedOnboardingContent.scss

```diff
@@ -4,6 +4,7 @@
   padding: 24px 0;
   a {
     color: #9247C2;
+    cursor: pointer;
```

Comment:
This isn't part of our default/baseline styles? :(

Reply:
I also assumed that would be the case but it was not showing up for me unless I added here. Its possible the react components are rendered outside the scope of whatever global styles the angular app provides.